src/main/scala/workshop4/01 Iterable.sc

```diff
@@ -28,3 +28,5 @@ def generateInfiniteCollection: Iterable[Double] = {
     }
   }
 }
+
```

Comment:
I'll remove those. I obviously was playing with the worksheet and left those newlines there by accident. I am still busy reviewing by the way, so hang on for updates to this PR.